File: wrappers/dotnet/indy-sdk-dotnet/WalletApi/WalletStorageException.cs
namespace Hyperledger.Indy.WalletApi
{
    /// <summary>
    /// Exception thrown when a storage error occurs during a wallet operation.
    /// </summary>
    public class WalletStorageException : IndyException
    {
        const string message = "A storage error occurred during the wallet operation.";

        /// <summary>
        /// Initializes a new WalletStorageException.
        /// </summary>
        internal WalletStorageException() : base(message, (int)ErrorCode.WalletStorageError)
        {

        }
    }

}

File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/wallet/WalletStorageException.java
package org.hyperledger.indy.sdk.wallet;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown occurred during wallet operation.
 */
public class WalletStorageException extends IndyException
{
	private static final long serialVersionUID = 1829076830401150667L;
	private final static String message = "Storage error occurred during wallet operation.";

	/**
	 * Initializes a new WalletStorageException.
	 */
	public WalletStorageException()
	{
		super(message, ErrorCode.WalletStorageError.value());
	}
}